configs/setups/fesom-recom/fesom-recom.yaml

```diff
@@ -95,91 +90,11 @@ fesom:
                                 runoff_ttp_ind: 3
                         forcing_reso:
                                 forcing_grid: 'T319'    #'T62','T319', 'other'
-                                forcing_dx: 0.4         #degree; from here required for 'other'
-                                forcing_dy: 0.4
-                                forcing_ni: 900         #number of grid points
-                                forcing_nj: 451
                         forcing_bulk:
                                 AOMIP_drag_coeff: true
-                namelist.ice:
-                        ice_stress:
-                                Pstar: 27500.0          #23000. 27500. 30000.
-                                c_pressure: 15.0        #20.0
-                        ice_fric:
-                                Cd_oce_ice: 1.0e-2      #5.5e-3  3.0e-3  !Qiang ab 1964
-                        ice_rheology:
-                                evp_rheol_steps: 300
-                        ice_scheme:
-                                ice_gamma_fct: 0.05
-                        ice_therm:
-                                Sice: 5.0
-                                h0: 0.6
                 namelist.oce:
                         viscdiff:
-                                Kh0: 800.0
-                                brine_rejection_param: false
-                        oce_scheme:
-                                gamma_fct: 0.4
-                        age_tracer:
-                                use_age_tracer: true
-                                age_tracer_start_year: "${age_tracer_sy}"
-                        tracer_cutoff:
-                                limit_salinity: false
-                                salinity_min: 5.0
```

Comment:
This changes take place only for `fesom-1.4-recom` simulations, and where needed to reproduce exactly the control runs provided by Judith a few years ago. Do we really want to change this settings an let fesom use the default fesom namelists defined in ESM-Tools for `fesom-recom`?

Reply:
Yes thats fine.